### PR TITLE
update to 1.26.15

### PR DESCRIPTION
**Pull request by M-Waszkiewicz-Anaconda**

### CODE REVIEW

File: recipe/meta.yaml [hunk 1]
```diff
@@ -1,13 +1,13 @@
...
 
 package:
   name: {{ name|lower }}
   version: {{ version }}
 
 source:
-  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/urllib3-{{ version }}.tar.gz
-  sha256: 076907bf8fd355cde77728471316625a4d2f7e713c125f51953bb5b3eecf4f72
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 8a388717b9476f934a21484e8c8e61875ab60644d29b9b39e11e4b9dc1c6b305
```
Comment (skupr-anaconda) [inline recipe/meta.yaml:12]: doc_source_url and license_url can be removed

Comment (skupr-anaconda) [reply recipe/meta.yaml:12]: pip install should be run with --no-deps.

Comment (M-Waszkiewicz-Anaconda) [reply recipe/meta.yaml:12]: done

**Review by skupr-anaconda — APPROVED** [review]

**Review by akabanovs — APPROVED** [review]
LGTM

**Pull request merged**